perf/evd/classic/evd_classic_perf.cpp

```diff
@@ -26,14 +26,16 @@ std::vector<std::string> tol_based_names = {
 int main() {
     size_t n = 4;
     std::vector<double> A = {7.0, 3.0, 2.0, 1.0, 3.0, 9.0, -2.0, 4.0, 2.0, -2.0, -4.0, 2.0, 1.0, 4.0, 2.0, 3.0};
-    std::vector<double> A_copy(n*n);
+    std::vector<double> A_copy(n * n);
     std::vector<double> e(n);
     std::vector<double> V(n * n, 0);
     matrix_t Data_matr = {&A[0], n, n};
     matrix_t Data_matr_copy = {&A_copy[0], n, n};
     vector_t E_vals = {&e[0], n};
     matrix_t E_vecs = {&V[0], n, n};
 
-    run_all(epoch_based_versions, epoch_based_names, Data_matr, Data_matr_copy, E_vecs, E_vals, 100);
-    run_all(tol_based_versions, tol_based_names, Data_matr, E_vecs, E_vals, 1e-8);
+    std::vector<double> costs(1, 10000);
```

Comment:
I don't quite understand why it was initialized to 10000.

I think we need to create something similar to the svd version, i.e. 

`    std::vector<double> costs;
    for (const auto& cost_fn : epoch_based_cost_fns) {
        costs.push_back(cost_fn(n, n, n_iter));
    }`

Will do it as soon as I calculate the flop count cost for the initial version.

Reply:
It is initialized to 10000 because I haven't calculated the EVD costs. You can update it with your own calculations.